var/spack/repos/builtin/packages/log4cxx/package.py

```diff
@@ -43,5 +43,8 @@ class Log4cxx(AutotoolsPackage):
     patch('log4cxx-0.10.0-narrowing-fixes-from-upstream.patch')
 
     def configure_args(self):
-        args = ['--disable-static']
+        args = ['--disable-static',
+                # 'apr''s 'bin' isn't pulled (via apr-util) into build env
+                '--with-apr={0}'.format(self.spec['apr'].prefix)]
```

Comment:
Does this actually work? If `apr` is a direct dependency, it should be listed in the dependencies above, not assumed to exist because it is a transitive dependency.